src/components/Announcement.tsx

```diff
@@ -51,18 +60,26 @@ export default function ({ redirectTo }: { redirectTo: string }) {
   const onPage = location.pathname === redirectTo
 
   return (
-    <div id="bottom-bar" className={announceWrapper(animate)}>
+    <div className={announceWrapper(animate)}>
       <div className={flex('flex-1')} />
-      <LinkText url={redirectTo}>
-        <AccentText small bold color="text-formal-accent">
-          Now introducing zk proofs for your work email!{' '}
-          {!onPage
-            ? 'Get started'
-            : account
-            ? 'Get started below.'
-            : 'Connect wallet to get started.'}
-        </AccentText>
-      </LinkText>
+      <div
+        className={classnames(
+          opacity(animate ? 'opacity-0' : undefined),
+          transitionProperty('transition-opacity'),
+          commonTransition
+        )}
+      >
+        <LinkText url={redirectTo}>
+          <AccentText small bold color="text-formal-accent">
+            Now introducing zk proofs for your work email!{' '}
+            {!onPage
```

Comment:
Let's correct the logic here:
```TS
onPage ? account ? 'Get started below.' : 'Connect wallet to get started' : 'Get started'
```
I think it's easier to read, you can extract central block like:
```TS
const hasAccount = account ? account ? 'Get started below.' : 'Connect wallet to get started'
onPage ? hasAccount : 'Get started'
```

Reply:
Thank you! Fixed